runtime/admin/src/main/java/org/apache/polaris/admintool/BootstrapCommand.java

```diff
@@ -71,6 +78,23 @@ static class FileInputOptions {
           description = "A file containing root principal credentials to bootstrap.")
       Path file;
     }
+
+    static class SchemaInputOptions {
+      @CommandLine.Option(
+          names = {"-v", "--schema-version"},
+          paramLabel = "<schema version>",
+          description = "The version of the schema to load in [1, 2, LATEST].",
+          defaultValue = SchemaOptions.LATEST)
+      String schemaVersion;
+
+      @CommandLine.Option(
+          names = {"--schema-file"},
+          paramLabel = "<schema file>",
+          description =
+              "A schema file to bootstrap from. If unset, the bundled files will be used.",
```

Comment:
"file" may be misleading here. The script is loaded from resources built into the admin tool jar, IIRC.

Reply:
I would actually like to modify that behavior; let me make some changes to support both resources on the classpath and arbitrary files to see what you think

Reply:
sgtm